diff -ur openssl-1.0.1e.orig/Configure openssl-1.0.1e/Configure
--- openssl-1.0.1e.orig/Configure	2013-11-15 13:39:57.000000000 +0100
+++ openssl-1.0.1e/Configure	2013-11-15 13:39:53.000000000 +0100
@@ -343,6 +343,10 @@
 ####
 #### Variety of LINUX:-)
 ####
+
+# Linux cross compile
+"linux-cross","gcc:-DTERMIO \$(EXTRA_CFLAGS) -fomit-frame-pointer -Wall::-D_REENTRANT::-ldl:BN_LLONG RC4_CHAR RC4_CHUNK DES_INT DES_UNROLL BF_PTR:${no_asm}:dlfcn:linux-shared:-fPIC::.so.\$(SHLIB_MAJOR).\$(SHLIB_MINOR)",
+
 # *-generic* is endian-neutral target, but ./config is free to
 # throw in -D[BL]_ENDIAN, whichever appropriate...
 "linux-generic32","gcc:-DTERMIO -O3 -fomit-frame-pointer -Wall::-D_REENTRANT::-ldl:BN_LLONG RC4_CHAR RC4_CHUNK DES_INT DES_UNROLL BF_PTR:${no_asm}:dlfcn:linux-shared:-fPIC::.so.\$(SHLIB_MAJOR).\$(SHLIB_MINOR)",
